addons/utm/models/utm_mixin.py

```diff
@@ -85,6 +85,11 @@ def _get_unique_names(self, model_name, names):
             Input: ['test', 'test [3]', 'bob', 'test', 'test']
             Output: ['test [2]', 'test [3]', 'bob', 'test [4]', 'test [5]']
 
+        The function also works case insensitively.
+        E.G.
+            Assume only UTM that exists in the database is "TeST"
+            Input: ['test', 'tesT [3]', 'bob', 'Test', 'test']
+            Output: ['test [2]', 'tesT [3]', 'bob', 'Test [4]', 'test [5]']
```

Comment:
Maybe we can just update the examples above to avoid adding a bunch of lines just for this

Reply:
@reth-odoo i will do in one commit, but that is confusing. because sometimes people tell me to have test and feature in different commits, sometimes in one, i don't get which situation is which

Reply:
I've never seen a PR with tests in a different commit. Not saying it can't happen but I don't see a reason for it here at least

Reply:
> @reth-odoo i will do in one commit, but that is confusing. because sometimes people tell me to have test and feature in different commits, sometimes in one, i don't get which situation is which

Would say: if you are going to change something that is quite important and it is currently not tested, sometimes it is a good idea to 1/ write tests (and commit), 2/ change a behavior and update tests. Otherwise we generally do both at the same time. Here I would say we don't need to separate commits between feature and tests.